pedalboard/io/AudioFileInit.h

```diff
@@ -155,15 +155,24 @@ inline void init_audio_file(
           "__new__",
           [](const py::object *, py::object filelike, std::string mode) {
```

Comment:
Doesn't apply to memoryviews and bytesIO, but [pybind11 will convert pathlib.Path objects to std::filesystem](https://pybind11.readthedocs.io/en/stable/advanced/cast/overview.html?highlight=path#list-of-all-builtin-conversions).

Though I am a little confused why you can't specialize this function on `py::buffer`